guides/spin/cli/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Install and Configure Spin CLI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Install `spin`
+
+One way to manage applications and pipelines as code is through [spin](http://github.com/spinnaker/spin).
+
+To acquire `spin`, do the following:
+
+```bash
+# See the versions released at https://github.com/spinnaker/spin/releases.
+export VERSION=<desired spin version tag> # Select a tag from the released version, e.g. v0.1.0
+curl -L https://github.com/spinnaker/spin/releases/download/$VERSION/spin -o spin
+chmod +x ./spin
+./spin --help
+```
+
+**Note**: The alpha `spin` binaries are only available for Linux platforms only at this point.
```

Comment:
Can delete "...at this point"

Can we get away with not indicating we'll support other platforms? I'd especially like to avoid mentioning a specific target Spinnaker version when we think we'll have them.